src/messages/GE.json

```diff
@@ -436,5 +436,9 @@
   "Delete collection": "Delete collection",
   "Leverage libraries": "Leverage the power of our client libraries. Download the",
   "Client library": "client library here ",
-  "SDKs documentation": "To read more about our SDKs, go to the documentation page at "
+  "SDKs documentation": "To read more about our SDKs, go to the documentation page at ",
+  "Add to collection": "Add to collection",
+  "Switch to beta": "Switch to beta",
+  "Resources": "Resources",
+  "Remove": "Remove"
```

Comment:
turns out there's actually a `remove` message in `GE.json` (on line 257) it's just that it's in lowercase letters